nncf/tensorflow/quantization/layers.py

```diff
@@ -34,6 +34,10 @@ def __init__(self, config: TFQuantizerSpec, data_format: str ='channels_last', *
         self._quantizer = self._create_quantizer(config, self._op_name)
         self._quantizer_weights = {}
 
+    @property
+    def quantizer(self) -> Quantizer:
```

Comment:
You introduced quantizer property to determine quantizer mode (symmetric, asymmetric). In this case, let's explicitly introduce the mode property instead of the quantizer property?

Reply:
Done